scripts/performance.ps1

```diff
@@ -278,6 +277,10 @@ function Run-Loopback-Test() {
     $serverOutput = $null
     try {
 
+        Write-Host $QuicPing
+        Write-Host $ServerArgs
+        Write-Host $ClientArgs
```

Comment:
I assume temporary? If so please remember to remove. If not, please use `Write-Debug`